ExchangeWebServices/Source/MS-OXWSCONT/TestSuite/TestSuiteBase.cs

```diff
@@ -798,6 +802,25 @@ protected void VerifyChildElementInContactItemTypeComplexType(ContactItemType re
                 47,
                 @"[In t:ContactItemType Complex Type] Birthday element: Represents the birthday of the contact (2).");
 
+            // Add the debug information
+            Site.Log.Add(LogEntryKind.Debug, "Verify MS-OXWSCONT_R4701");
+
+            // Verify MS-OXWSCONT requirement: MS-OXWSCONT_R4701
+            Site.CaptureRequirementIfIsTrue(
+                responseContactItem.BirthdayLocalSpecified,
+                4701,
+                @"[In t:ContactItemType Complex Type] The type of the element of BirthdayLocal is xs:dateTime.");

```

Comment:
Missing [XMLSCHEMA2] at the end.

Reply:
'BirthdayLocal	xs:dateTime	Represents the birthday of the contact (2) in the client's local time zone.<6>'
There is no [XMLSCHEMA2] in the original text.